locale/fi/LC_MESSAGES/django.po

```diff
@@ -1029,6 +1029,15 @@ msgstr "Per viikko"
 msgid "Fixed"
 msgstr "Kiinteä"
 
+msgid "Inside"
+msgstr "Sisätila"
+
+msgid "Outside"
+msgstr "Ulkotila"
+
+msgid "Inside and outside"
+msgstr "Ulko- ja sisätila"
```

Comment:
Are you sure about all these translations as "Sisätila" would be "Interior space" etc.

Reply:
This came from client :)